netlify-plugin-cloudinary/src/index.ts

```diff
@@ -277,7 +279,7 @@ export async function onBuild({
           }
 
           mediaPaths.forEach(async mediaPath => {
-            const cldAssetPath = `/${path.join(PUBLIC_ASSET_PATH, mediaPath)}`;
+            const cldAssetPath = `/${path.posix.join(PUBLIC_ASSET_PATH, mediaPath)}`;
```

Comment:
just curious about the context of this change. im not familiar so had to look it up, and looks like it makes the `.join` specific to unix environments, does that sound right?

were you running into an issue? im not quite familiar enough to know about the scope of this, but i mostly only wonder how this would impact anyone on windows trying to test this project locally as i would imagine any Netlify environment would be posix

Reply:
based on some of the fixes i see in the test suite, guessing thats the case. curious though on what the issue you ran into is, maybe you're on windows?

Reply:
Howdy! I tried my best to provide a description in the related commit, but tldr: 

- `path.join()` is meant for filesystem paths; it uses `path.sep` to join
- I'm on Windows, `path.sep === \`
- `getCloudinaryUrl()` returns a to URL containing a url encoded backslash `%5C` causing `Redirects` tests in `on-build.test.js` to fail
- Using `path.posix.join()` means we will always use a forward slash `/` to create the URI

Here's the [SO answer that gave me the `path.posix.join()` suggestion](https://stackoverflow.com/a/54368132).

Reply:
apologies, i didnt look at the commit messages 🙈 ill look there first next time before asking

very interesting, never hit that before! thanks

Reply:
No worries at all, I'm glad you asked here because it gave me the opportunity to elaborate and provide visuals! 🚀